### DISCUSSION TITLE
Installing custom packages via pip overwrites existing packages

### DISCUSSION THREAD

**Discussion by ekawatani** [custom-components/pyscript#345] [General]
I'm running HA OS on Raspberry Pi.

I installed a package pyhton-switchbot via pip inside the  docker container because I needed it for my scripts. However, this overwrote the 'switchbot' package that's already installed for the official Switchbot integration. As a result, I could not install the integration anymore because the switchbot package now has different APIs.

What is the correct way to install third party packages for the use with pyscript? Do virtualenv work?

**Comment by SiboutVanLoo** [comment]
Do you have any news on this question?
I might be looking for the same thing soon. :-)
